app/server.py

```diff
@@ -63,6 +63,8 @@ def search(search_engine):
         if db['queries'].find({query: query}).limit(1) is False:
```

Comment:
Need a change here, to consider not only query, but the engine as well.

Reply:
Agree 👍